homeassistant/components/steam_online/sensor.py

```diff
@@ -140,10 +147,28 @@ def _get_current_game(self):
         _LOGGER.error("Unable to find name of app with ID=%s", game_id)
         return repr(game_id)
 
+    def _get_last_online(self):
```

Comment:
Docstring is missing.

Reply:
Added Docstring

Reply:
@fabaff please let me know what you think of the changes or if there is anything else pending on this PR. Thanks